server/src/main/kotlin/delta/codecharacter/server/user/public_user/PublicUserService.kt

```diff
@@ -71,11 +69,50 @@ class PublicUserService(@Autowired private val publicUserRepository: PublicUserR
                     wins = it.wins,
                     losses = it.losses,
                     ties = it.ties
-                )
+                ),
             )
         }
     }
 
+    fun updateLeaderboard(leaderboardList: List<LeaderboardEntryDto>) {
```

Comment:
This is a scheduled function. A scheduled function can not accept parameters by default.